internal/log/zap.go

```diff
@@ -0,0 +1,70 @@
+package log
+
+import (
+	"io"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var zapLogger *ZapLogger
+
+type ZapLogger struct {
+	*zap.SugaredLogger
+}
+
+// SetOutput...
+// To satisfy log.Interface
+func (z *ZapLogger) SetOutput(io.Writer) {
+	return
```

Comment:
Interesting. Does zap not support changing the output location?

Reply:
You can set the output location within the zap config before instantiating the logger. See [here](https://github.com/iwpnd/tegola/blob/a55b7fb9eec551e28befaa6551c2152d1a6e29fc/internal/log/zap.go#L57).

```
zapconfig.OutputPaths = []string{
    "/var/log/myproject/myproject.log",
  }
```

would also be possible here.

Reply:
I think we should drop SetOutput from the Interface{}, we don't have any calls to it, and having the output set during the configuration of the logger makes the most sense.